### ISSUE TITLE
Add entity browser table form widget to featured content paragraph

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#414]
### Problem
The current entity browser field on featured content displays selected content in a draggable well similar to media image content. This UX isn't consistent with other entity reference multiple fields and can be confusing to users with regard to what order the content is in.

This is a follow-up to #220 and #337.

### Proposed Solution
Blocked waiting on #337 and the D.o issue below.

Watch https://www.drupal.org/project/entity_browser/issues/2973457 and implement it as soon as the patch is more stable or merged in.

It provides an additional table widget which will have to be changed/configured/exported on the form display of the featured content paragraph.

### Test
Make sure the table form widget renders correctly nested with our paragraphs solution, changes to order are saved and the remove buttons for each piece of content are still available/function correctly.

Other sites should receive this new configuration without data loss.

**Comment by joewhitsitt** [comment]
This was working the last time I tested (patch 153).

**Comment by joewhitsitt** [comment]
Closing as a won't do since this functionality is only on v2. v3 is just using a normal entity reference field.
